packages/trader/src/Modules/Contract/Components/AccumulatorsStats/ticks-history-counter.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type TTicksHistoryCounter = {
```

Comment:
btw i have my own TS migrated version and test in my PR lol

https://github.com/binary-com/deriv-app/pull/8852/files#diff-8d0b9ba6bf53291764b8e53c440e81d09e14a00153c1b03ee0288e83e46abe47

Reply:
same with some of the other files

Reply:
@henry-deriv I migrated this file to TS in the first place only because you asked me to earlier here: https://github.com/binary-com/deriv-app/pull/8200#discussion_r1223911997 If there will be conflicts, we will fix them :)

Reply:
coo coo